notebooks/plotly_roc_plot.py

```diff
@@ -97,9 +114,9 @@
 
 if args.hisp_yellow:
     colorscale_names = {
-        "ACDC": "Blues_r",
+        "ACDC": "Purp",
         "SP": "Greens_r",
-        "HISP": "pinkyl",
+        "HISP": "YlOrBr",
```

Comment:
I would like to standardize on "Darker is higher" or "Darker is lower" and that's not used here